tools/dockerfile/distribtest/csharp_ubuntu1604_x64/Dockerfile

```diff
@@ -17,11 +17,16 @@ FROM ubuntu:16.04
 RUN apt-get update && apt-get install -y \
     mono-devel \
     ca-certificates-mono \
-    nuget
+    nuget \
+    && apt-get clean
 
 # make sure we have nuget 2.12+ (in case there's an older cached docker image)
 RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
 RUN echo "deb http://download.mono-project.com/repo/debian wheezy main" | tee /etc/apt/sources.list.d/mono-xamarin.list
-RUN apt-get update && apt-get install -y nuget
+RUN apt-get update && apt-get install -y nuget && apt-get clean
```

Comment:
nit: Can we add the mono-xamarin apt source as the first thing in this docker file, and avoid installing nuget for a second time here?

Reply:
I wanted to install  the mono version that comes natively with ubuntu16.04.
The nuget upgrade is only a fix applied on top of the vanilla mono installation (and I'm hoping that one day we will be able to get rid of it).